rca/project_styleguide/templates/patterns/organisms/programme_tabs/curriculum.html

```diff
@@ -30,6 +30,22 @@ <h2 class="section__heading section__heading--secondary heading heading--five">{
             </div>
         </div>
     {% endif %}
+
+    {% if page.quote_carousel %}
+        <div class="section__row section__row--first-extra-small section__row--last-extra-small">
+            {% include "patterns/organisms/carousel/carousel--quotes.html" with carousel=page.quote_carousel modifier='carousel--square carousel--quotes carousel--no-margin' control_title="Quote carousel" datatag='data-peek-carousel' %}
+        </div>
+        {% if page.quote_carousel_link %}
+            <div class="section__row section__row--last-small grid">
+                <div class="layout__@medium-start-one layout__@large-start-two layout__span-one">
+                    <a class="link link--secondary link--link" href="{{ page.quote_carousel_link }}">
+                        <span class="link__label">Browse our student's experience's</span>
```

Comment:
This looks like a typo -> "Browse our students' experiences"

Reply:
I've updated this.